modules/auxiliary/client/smtp/emailer.rb

```diff
@@ -161,7 +159,8 @@ def run
 
       if zip_payload
         zip_file = attachment_file.sub(/\.\w+$/, '.zip')
```

Comment:
Is this a reasonable suggestion? (as per https://github.com/rapid7/metasploit-framework/pull/11310#issuecomment-457337672)

```suggestion
        zip_file = "#{attachment_file}.zip"
```